File: config/config-sbc2ha/src/main/java/com/dfi/sbc2ha/config/sbc2ha/definition/sensor/oneWire/therm/DS18B20.java
package com.dfi.sbc2ha.config.sbc2ha.definition.sensor.oneWire.therm;

import com.dfi.sbc2ha.config.sbc2ha.definition.enums.PlatformType;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public abstract class DS18B20 extends OneWireTherm {

    public DS18B20() {
        super();
        platform = PlatformType.DS2482;
    }
}

File: config/editor/src/app/definition/sensor/oneWire/therm/DS18B20.ts
import {OneWireTherm} from "./OneWireTherm";
import {PlatformType} from "../../../enums/PlatformType";

export abstract class DS18B20 extends OneWireTherm {

  constructor(platform: PlatformType, name: string, busId: string, address: string) {
    super(platform, name, busId, address);
  }
}
